addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/converter/swagger/RequestModelConverter.java

```diff
@@ -88,4 +102,39 @@ private String generateBody() {
         }
         return "";
     }
+
+    private String extractExampleBody(Content content, String mediaTypeString) {
```

Comment:
This should be part of `BodyGenerator` (which already uses examples in some cases).